model/event/event_test.go

```diff
@@ -411,7 +411,7 @@ func (s *eventSuite) TestTaskEventLogLegacyEvents() {
 func (s *eventSuite) TestFindLastProcessedEvent() {
 	events := []EventLogEntry{
 		{
-			ID:           bson.NewObjectId(),
+			ID:           bson.NewObjectId().String(),
```

Comment:
All your calls to .String() should be .Hex()

Reply:
oops, fixed